wazo_sysconfd/plugins/systemd_services/services.py

```diff
@@ -4,27 +4,12 @@
 import logging
 import subprocess
 
-from xivo import http_json_server
-from xivo.http_json_server import HttpReqError
-from xivo.http_json_server import CMD_RW
+from . import exceptions
 
 logger = logging.getLogger('wazo_sysconfd.modules.services')
 
 
-class InvalidActionException(ValueError):
-    def __init__(self, service_name, action):
-        super(InvalidActionException, self).__init__(self)
-        self.service_name = service_name
-        self.action = action
-
-
-class InvalidServiceException(ValueError):
-    def __init__(self, service_name):
-        super(InvalidServiceException, self).__init__(self)
-        self.service_name = service_name
-
-
-def services(args, options):
+def services(args):
```

Comment:
There are other modules depending on `wazo_sysconfd.modules.services`. Please update them accordingly.

Reply:
Done, the only call I found is within `xivoctl` module